sp800-63b/references.md

```diff
@@ -63,52 +63,52 @@
 
```

Comment:
RFC number a document identifier like an SP number. I have no opinion on how/whether to include DOIs, just noting that we do that differently IETF from others.